app/models/policy.rb

```diff
@@ -27,8 +27,6 @@ class Policy < ApplicationRecord
   after_update :destroy_orphaned_business_objective
   after_rollback :destroy_orphaned_business_objective
 
-  after_save :update_counters!
```

Comment:
why?

Reply:
Because this is only called via `Policy#update` and was only being used in tests.